client-gateway/bot/passenger.py

```
@@ -1,40 +1,54 @@
-import logging
-from telebot import types
-
-logger = logging.getLogger('drive_ops')
-
-
-def register_handlers(bot, user_orders, user_roles, buttons, keyboards, helpers):
->>>>>>> main
+        # ...existing code and nested function definitions...
+        # Register check status conversation handler and refresh callback at the end
+    check_status_handler = ConversationHandler(
+        entry_points=[MessageHandler(filters.Regex(f"^{re.escape(BTN_CHECK_STATUS)}$"), start_check_status_flow)],
```

Flake8 (7.3.0)

[error] undefined name 'BTN_CHECK_STATUS'

(F821)